02-peer-protocol.md

```diff
@@ -863,6 +869,10 @@ seconds, and the protocol only supports an expiry in blocks.
 specification; larger amounts are not necessary, nor wise, during the
 bootstrap phase of the network.
 
+The node _responsible_ for paying the Bitcoin fee should be allowed to dip into
+its channel reserve to accommodate a fee increase, otherwise the channel may end
+up in an unusable state (see [#728](https://github.com/lightningnetwork/lightning-rfc/issues/728)).
```

Comment:
How do nodes signal that they allow this? Older nodes wouldn't allow this.

Reply:
I don't think you need to signal anything.

First case: sender is upgraded, receiver is not. The sender will send an HTLC that the receiver will reject (a non-upgraded sender wouldn't have sent that HTLC - the channel would be stuck). No big deal.

Second case: receiver is upgraded, sender is not. The sender will not send any HTLC, so the channel will be stuck. It's still possible to restore the channel by using below dust HTLCs until it's unblocked.